src/main/java/com/iesam/digLibrary/features/loans/presentation/LoanPresentation.java

```diff
@@ -68,10 +68,10 @@ public static void deleteLoan(){
     }
     public static void returnResourceToEndLoan(){
         System.out.println("Introduce the resource ID you are returning: ");
```

Comment:
Instead of 'Introduce' you could use 'Insert', althought is good that way too. 😊

Reply:
Thanks for the feedback 😄